package.json

```diff
@@ -25,6 +25,31 @@
     "testcases-assets-dir": "test-assets/",
 	  "testcases-export-dir": "auto-wcag-testcases/",
 	  "testcases-export-filename": "testcases.json",
-	  "rules-export-filename": "rules.json"
+    "rules-export-filename": "rules.json",
+    "@context": {
+      "earl": "http://www.w3.org/ns/earl#",
+      "xlink": "http://www.w3.org/1999/xlink/",
+      "tcdl": "http://bentoweb.org/refs/TCDL2.0/",
+      "a11y-testcases": {
+        "@id": "tcdl:testCaseDescription",
+        "@container": "@list"
+      },
+      "url": "https://schema.org/URL",
```

Comment:
This somehow has to map to a `tcdl:file` with an `xlink:href` attirbute. It may be that the output will have to be adjusted slightly for that to work though, e.g.:

```json5
{
  "@context": {
    "file": {
      "@id": "tcdl:file"
    },
    "fileUrl": "xlink:href"
  },
  "a11y-testcases": [
    {
      ...
      "file": {
        "fileUrl": "https://auto-wcag.github.io/auto-wcag/rules/..."
      },
      ...
    }
  ]
}
```

@nitedog Might you have any idea if this can be done in a less verbose manner?